airbyte-integrations/connectors/source-plausible/integration_tests/configured_catalog.json

```diff
@@ -10,4 +10,4 @@
       "destination_sync_mode": "overwrite"
     }
   ]
-}
+}
```

Comment:
Please remove the file from this PR as it is unrelated to the LinkedIn Ads source connector.